api/src/main/java/edu/cornell/mannlib/vitro/webapp/servlet/setup/ContentModelSetup.java

```diff
@@ -298,7 +298,7 @@ private boolean applyChanges(Model baseModel, Model newModel, Model userModel, S
 
             if (!difOldNew.isEmpty()) {
                 difOldNew.write(out, "TTL"); 
-                log.debug("Difference for " + modelIdString + " (old -> new), these triples should be removed: " + out.toString());
+                log.debug("Difference for " + modelIdString + " (old -> new), these triples should be removed: " + out.toStringtoString());
```

Comment:
This seems twice as problematic as a single `toString`.

Reply:
i am sorry, don't know what happend there and did not check the commit adequately. I removed the mistake